omglol/models.go

```diff
@@ -145,6 +145,12 @@ type dnsChangeResponse struct {
 	} `json:"response"`
 }
 
+type Paste struct {
+	Title      string `json:"title"`
+	Content    string `json:"content"`
+	ModifiedOn int64  `json:"modified_on"`
```

Comment:
I noticed in the API (not yet documented), that there is also now a `listed` attribute, much like with PURLs.
```suggestion
	ModifiedOn int64  `json:"modified_on"`
	Listed     bool   `json:"listed"`
```